src/python/pants/engine/subsystem/native_engine_version

```diff
@@ -1 +1 @@
-e5b11346a458240934cc6136e40faa62b75f79ff
+da39a3ee5e6b4b0d3255bfef95601890afd80709
```

Comment:
I think the only thing I did was a fetch and merge, and then run engine testing. Thus this sha does look like the latest one. Is it possible that in ci we are using the latest binary?